src/main/java/org/graylog/integrations/aws/AWSPermissions.java

```diff
@@ -0,0 +1,36 @@
+package org.graylog.integrations.aws;
+
+import com.google.common.collect.ImmutableSet;
+import org.graylog2.plugin.security.Permission;
+import org.graylog2.plugin.security.PluginPermissions;
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.graylog2.plugin.security.Permission.create;
+
+public class AWSPermissions implements PluginPermissions {
```

Comment:
You have to register the custom permissions in the `IntegrationsModule` with `addPermissions(AWSPermissions.class)`. Otherwise they will not show up in the REST endpoints that return all available permissions.

Reply:
I know that @ceruleancee had done this. Perhaps the file did not get committed. I committed this change now, so this PR can be merged. 🙂